packages/lib-react-components/src/ZooHeader/ZooHeader.js

```diff
@@ -67,14 +67,14 @@ export default function ZooHeader({
 
   return (
     <StyledHeader
+      as='header'
       background='black'
       direction='row'
       fill='horizontal'
       justify='between'
       pad='none'
       responsive={false}
       role='presentation'
```

Comment:
Why has this header got a 'presentation' role?

Reply:
Aha, [Using Navigation Landmarks](https://accessibility.blog.gov.uk/2016/05/27/using-navigation-landmarks/): a page should only have one `header`, which is a direct child of `body`. I assume this gives the page more than one header, in which case maybe remove the `header` here, or restructure the page so that it only has one header, as per the advice in Léonie Watson's post.

Reply:
Oh gods, we have **six** `header`s on a project home page like Planet Hunters TESS, only one has role=presentation.

I dove a bit into understanding how `header`s should used (issue: #3710) and am in a bit of a rabbit hole; I think my next step is to actually learn to properly use my Mac VoiceOver so I can understand how people with screenreaders experience our website, instead of just trying my best to understand the HTML & ARIA specs.